spvc/test/run_spirv_cross_tests.py

```diff
@@ -45,127 +43,112 @@
 test_count = 0
 pass_count = 0
 
-not_used, tmpfile = tempfile.mkstemp()
-devnull = open(os.devnull, 'w')
-
-def mkdir_p(filepath):
-    ''' make sure directory exists to contain given file '''
-    dirpath = os.path.dirname(filepath)
-    try:
-        os.makedirs(dirpath)
-    except OSError as exc:
-        if not (exc.errno == errno.EEXIST and os.path.isdir(dirpath)):
-            raise
+def check_call(cmd):
+    global devnull
+    if subprocess.call(cmd, stdout=devnull):
+        print('failed command: ', ' '.join(cmd))
+        sys.exit(1)
```

Comment:
Either return an error or throw an exception.

Reply:
Whoops, that was a debugging print I meant to remove.
This will call subprocess.check_call and throw an exception.